src/client/index.ts

```diff
@@ -197,40 +197,49 @@ export class Client {
    *   as a string with a time unit (e.g. "1h", "30m", "15s").
    * @returns workflow run id
    */
-  public async trigger({
-    url,
-    body,
-    headers,
-    workflowRunId,
-    retries,
-    flowControl,
-    delay,
-    failureUrl,
-    useFailureFunction,
-  }: TriggerOptions): Promise<{ workflowRunId: string }> {
-    failureUrl = useFailureFunction ? url : failureUrl;
+  // Overload signatures
```

Comment:
can you remove this comment and update the docstring above with an example of batch?